src/include/sof/boot_test.h

```diff
@@ -6,16 +6,9 @@
 #ifndef __SOF_BOOT_TEST_H__
```

Comment:
typos: initialisation -> initialization, raices -> races

Reply:
"initialisation" isn't a typo, at least according to my dictionary, it's a "wrong" language (variant) ;-) The other one is indeed an error.